recipes/libjpeg/all/conanfile.py

```diff
@@ -54,7 +58,7 @@ def _settings_build(self):
         return getattr(self, "settings_build", self.settings)
 
     def build_requirements(self):
-        if self._settings_build.os == "Windows" and not self._is_msvc and \
+        if self._settings_build.os == "Windows" and not (self._is_msvc or self._is_msvc) and \
```

Comment:
```suggestion
        if self._settings_build.os == "Windows" and not (self._is_msvc or self. _is_clang_cl) and \
```

typo?

Reply:
BTW, can use now the official is_msvc from tools